bsp/wch/risc-v/yd-ch32v307vct6/applications/main.c

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) 2024, GSunwinder
```

Comment:
？

Reply:
I don't understand the question, but if you want to change the license, you can install the correct one.